jumpcutter/__main__.py

```diff
@@ -121,19 +131,14 @@ def main() -> None:
         args.space_on_edges,
     )
     for cut_type, jumpcutted_clip in outputs.items():
-        if len(outputs) == 2:
-            jumpcutted_clip.write_videofile(
-                str(
+        if len(outputs) == 1:
```

Comment:
The original code modified the `output_path` when `len(outputs) == 2`. Why do we have `==1` now?